systemd/authd.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=Authd daemon service
+After=authd.socket
+Requires=authd.socket
+PartOf=authd.socket
+
+[Service]
+Type=notify
+ExecStart=/sbin/authd
+
+# Some daemon restrictions
+CapabilityBoundingSet=
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=true
+PrivateDevices=yes
+PrivateNetwork=yes
+PrivateTmp=yes
```

Comment:
Also `PrivateMounts` maybe?

Reply:
It should not be a problem, since the daemon configuration data is in var.

Also `PrivateMounts` seems to be implied when `PrivateNetwork` is on, so it *should* already active. I'd use it explicit though :).

---

However, going back to the configuration data... Should we set `StateDirectory=authd` (and maybe `StateDirectoryMode=0700`)?

So that `/var/lib/authd` is always authorized.

Reply:
Yeah, we need to check this, but it's definitely the path that authd should use for any stored data.

Reply:
Fair enough. We're good to go from my POV.